mimic/model/clb_objects.py

```diff
@@ -13,6 +13,8 @@
 
 from six import string_types
 
+from toolz.dicttoolz import dissoc
+
```

Comment:
This library contains simple useful pure functions. otter has been using it for 5-6 months now. I can remove it if you are not comfortable with it.